lib/api/redfish-1.0/systems.js

```diff
@@ -779,7 +780,40 @@ var getSimpleStorage = controller(function(req, res)  {
             }).catch(function(error) {
                 return redfish.handleError(error, res);
             });
-        } else {
+        } else if(result.vendor === 'Cisco'){
+            return dataFactory(identifier, 'UCS:board')
+                .then(function(catalogsData){
+                    var controllers = [];
+                    switch(index.split('_')[0])  {
```

Comment:
Suggest use RegExp instead of enumerating method.
Please coordinate with Kevin to update this.

Reply:
not need to use RegExp